package.json

```diff
@@ -13,8 +13,8 @@
     "eslint": ">=6.6.0"
   },
   "dependencies": {
-    "@typescript-eslint/eslint-plugin": "^2.34.0",
-    "@typescript-eslint/parser": "^2.34.0",
+    "@typescript-eslint/eslint-plugin": "^4.33.0",
+    "@typescript-eslint/parser": "^4.33.0",
```

Comment:
Can't we still use version 5?

Reply:
`@typescript-eslint/*` v5 will be included when going to ESLint v8

Reply:
I believe `@typescript-eslint/*` v5 supports ESLint v6. I don't think we should wait for ESLint v8.
https://github.com/typescript-eslint/typescript-eslint/blob/da597c7177ae2692c0e1a4c61f2498b5871d4d09/packages/eslint-plugin/package.json#L70